tools/go-cli/go-whisk-cli/commands/namespace.go

```diff
@@ -78,7 +78,7 @@ var namespaceGetCmd = &cobra.Command{
         namespace, _, err := client.Namespaces.Get(qName.namespace)
         if err != nil {
             whisk.Debug(whisk.DbgError, "client.Namespaces.Get(%s) error: %s\n", qName.namespace, err)
-            errStr := fmt.Sprintf("Unable to obtain namespace entities for namespace '%s': %s", qName.namespace, err)
+            errStr := fmt.Sprintf("Unable to obtain namespace entities for namespace '%s'", qName.namespace)
```

Comment:
@rabbah , is the latest change expected, shall I change the test code to make the CI pass?